src/main/kotlin/org/ozwillo/dcimporter/service/ProcessingStatService.kt

```diff
@@ -0,0 +1,178 @@
+                processingStatRepository.findByModelAndCreationDateAfter(model, stringToLocalDateTime(date))
+            //date+organization
+            !date.isEmpty() && model.isEmpty() && !organization.isEmpty() && action.isEmpty() ->
+                processingStatRepository.findByOrganizationAndCreationDateAfter(organization, stringToLocalDateTime(date))
+            //date+action
+            !date.isEmpty() && model.isEmpty() && organization.isEmpty() && !action.isEmpty() ->
+                processingStatRepository.findByActionAndCreationDateAfter(action, stringToLocalDateTime(date))
+            //date+model+organization
+            !date.isEmpty() && !model.isEmpty() && !organization.isEmpty() && action.isEmpty() ->
+                processingStatRepository.findByModelAndOrganizationAndCreationDateAfter(model, organization, stringToLocalDateTime(date))
+            //date+model+action
+            !date.isEmpty() && !model.isEmpty() && organization.isEmpty() && !action.isEmpty() ->
+                processingStatRepository.findByModelAndActionAndCreationDateAfter(model, action, stringToLocalDateTime(date))
+            //date+organization+action
+            !date.isEmpty() && model.isEmpty() && !organization.isEmpty() && !action.isEmpty() ->
+                processingStatRepository.findByOrganizationAndActionAndCreationDateAfter(organization, action, stringToLocalDateTime(date))
+            //model+organization
+            date.isEmpty() && !model.isEmpty() && !organization.isEmpty() && action.isEmpty() ->
+                processingStatRepository.findByModelAndOrganization(model, organization)
+            //model+action
+            date.isEmpty() && !model.isEmpty() && organization.isEmpty() && !action.isEmpty() ->
+                processingStatRepository.findByModelAndAction(model, action)
+            //model+organization+action
+            date.isEmpty() && !model.isEmpty() && !organization.isEmpty() && !action.isEmpty() ->
+                processingStatRepository.findByModelAndOrganizationAndAction(model, organization, action)
+            //organization+action
+            date.isEmpty() && model.isEmpty() && !organization.isEmpty() && !action.isEmpty() ->
+                processingStatRepository.findByOrganizationAndAction(organization, action)
+            //model+organization+action+date
+            !date.isEmpty() && !model.isEmpty() && !organization.isEmpty() && !action.isEmpty() ->
+                processingStatRepository.findByModelAndOrganizationAndActionAndCreationDateAfter(model, organization, action, stringToLocalDateTime(date))
+            else -> processingStatRepository.findAll()
+        }
+    }
+
+    fun getByDate(date: LocalDateTime): Flux<ProcessingStat> {
+        return processingStatRepository.findByCreationDateAfter(date)
+    }
+
+    fun getGeneralStat(date: String): Mono<ProcessingResumeFields>{
```

Comment:
you'are doing a lot of things in this method but that's ok for this very first iteration